lms/views/content_item_selection.py

```diff
@@ -103,6 +85,6 @@ def content_item_form(request, lti_params, lms_url, content_item_return_url, jwt
         "jwt": jwt,
     }
 
-    if should_show_file_picker(lti_params, request):
+    if canvas_files_available(request, params=lti_params):
```

Comment:
Can you remind me what `lti_params` are here and why these can't be fetched from the request?

Reply:
They're the `request.params` from a _previous_ LTI launch request, that were stashed in the DB before redirecting the iframe to Canvas's authorization page, and now (as we're handling Canvas's redirect of the iframe back to us) they've been retrieved from the DB again.